content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "custom-rom-101"
 date = "2018-02-25T00:00:00"
+image = "img/rohanvtk/csrom.jpg"
```

Comment:
Can you optimize your header image? 812 Kb ain't a lot, but considering the size of the repo is already at about 16 mb. Hence, we have to optimize what we can. You can also use URL's if you wish too.